### DISCUSSION TITLE
Comunicação com a APyB - PySul24

### DISCUSSION THREAD

**Discussion by talitarossari** [apyb/comunidade#108] [Geral]
Opa pessoal, tudo certo?
Estamos começando nossos trabalhos que demandam aprovações, assinaturas e pagamentos da APyB.
Queríamos saber se vocês acham necessário um grupo de sincronização entre a Python Sul e a APyB.

Caso não achem necessário, deixo aqui algumas dúvidas relacionado a assinatura de contratos:

- Vocês tem alguma doc de procedimento para assinatura de contratos em nome da APyB?
- Há um formulário de "pedido de assinatura", como o de pedido para gerar a nota fiscal?
- Vocês recomendam/utilizam alguma plataforma de assinatura digital?

Agradeço imensamente desde já!

**Comment by anicelysantos** [comment]
Oi, Talita! :)

O ideal é que todos utilizem o caminho oficial para demandas que está escrito no [manual de como organizar eventos utilizando a infraestrutura da ApyB](https://apyb.python.org.br/guias/como-organizar-eventos/). Criar grupos pode causar ruídos no processo. 

A assinatura pode ser feita pela responsável do evento, desde que seja membro da Apyb. Só em uma exigência muito pontual é que nós assinamos. Você pode utilizar a plataforma que se sentir mais confortável (gov.br, Docsing, Clicksign, etc).

**Comment by talitarossari** [comment]
Opa @anicelysantos, perfeito então! Muito obrigada <3
